src/screens/MemberDetail/MemberDetail.spec.tsx

```
@@ -492,6 +739,181 @@
+      data: { deleteUser: { id: 'rishav-jha-mech' } },
+    });
+
+    const deleteUserWithoutData = vi.fn().mockResolvedValue({
+      data: null,
+    });
+
+    // Spy on toast.success
+    const toastSuccessSpy = vi.spyOn(toast, 'success');
+
+    // Test variables
+    let mockHandleDeleteUser: () => {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)